### DISCUSSION TITLE
Do I understand Previous Value correctly.

### DISCUSSION THREAD

**Discussion by riegera2412** [jomjol/AI-on-the-edge-device#2229] [General]
Hey there,

I have an issue understanding previous value or better its retention after a reboot. From the documentation I conclude the last PV will be retained after a reboot. However, each time after a reboot it isnt and replaced by the next error free digitization. Since my water meter is currently close to 301.XYZ the "1" is always recognized as a 7 which gives an increase of 6 m³, which is way to high and would be recognized as an error if the PV before the reboot would be retained. 

So the question is, is my understanding of PV retention over a reboot correct and this is some sort of bug or did I just concluded incorrectly?

Thanks in advance for the clarification.
A.

**Comment by caco3** [comment]
yes, it should be keep beyond a reboot by default.
Do you have [PreValueUse](https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-prevalueuse) enabled ?
And is [PreValueAgeStartup](https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-prevalueagestartup) on  a reasonable value?

See also https://jomjol.github.io/AI-on-the-edge-device-docs/FAQs/#pre-value

**Reply by riegera2412** [reply]
Hey there

and thank for the reply. Yes they are both set. PreValueUse is set to true via the interface and PreValueAgeStartup is set to a day (1440 minutes), sinces water usage is quite small and the highest peak only occurs once a day.

PS: In the link you've given, I just saw that the Consistency Check is only performed for the dig-100-models? Is that correct, since I have it on and been using the dig-cont-models for at least 3 months. If the Consistency Check doesnt do anything I would just turn it off.

**Reply by ghost** [reply]
Consistency Check is for class10 only. The newer models do not need such compensation.
